index.html

```diff
@@ -325,5 +325,21 @@ <h2 class="kn_1">
     </li>
   </ul>
 </section>
+    <footer id="contact" class="contact">
+      <div class="container-contact">
+        <h2 class="first-text-contac"> Contact me</h2>
+        <p class="second-text-contact">If you have an application you are interested in developing, a feature that you need built or a
+            project that
+            needs coding. I’d love to help with it </p>
+      </div>
+          
+      <form class="container-form" action="https://formspree.io/f/xgedzbjq" method="post" novalidate>
+        <input id="name" class="text-area" type="text" name="name" maxlength="30" required placeholder="name">
+        <input id="email" class="text-area" type="email" name="email"  required
+          placeholder="email">
+        <textarea id="message" class="text-area-large" placeholder="Write your message" maxlength="500" required></textarea>
+        <input class="btn-container-contact" type="submit" name="submit" value="Get In Touch">
+      </form>
+    </footer>
```

Comment:
Kindly try to match the Figma design for the contact form:
- [ ] Kindly add the same `background image` in the Figma design.
- [ ] Please make the `contact me` header in the middle of the container and change its color to match the Figma design.
- [ ] Please make the inputs field of the same width.
- [ ]  Kindly decrease the left padding for each input.
- [ ]  Please make the `Get In Touch` button on the left and increase padding for it to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/159698185-0f585602-ccbf-4fac-a7e5-c99612890110.png) | ![image](https://user-images.githubusercontent.com/24703171/159698509-b2b37de1-4225-4048-9da0-b4dc1f42ee5d.png) |